sirepo/sim_data/__init__.py

```diff
@@ -622,6 +622,26 @@ def schema(cls):
     def sim_file_basenames(cls, data):
         return cls._sim_file_basenames(data)
 
+    # TODO(mvk):  use a server to provide access to sim db and library files
+    @classmethod
+    def _sim_file_to_other_sim_lib(
```

Comment:
Make this an srw specific method `srw_sim_file_to_other_sim_lib` and put in sim_data.srw

Reply:
[987d29e](https://github.com/radiasoft/sirepo/pull/5216/commits/987d29e2fd3306bb306756b8fe3228dad1047030)

I'm still not sure of this - I think I should use copy and not a GET request, but it's not clear to me...

Reply:
[69fc7bc](https://github.com/radiasoft/sirepo/pull/5216/commits/69fc7bc86e07aabb4df966c0acf8a334a5668c8e)
This is better (is it?)

Reply:
This is template code (see also https://github.com/radiasoft/sirepo/issues/5254 because there is no qcall here), this won't work right. It would need be a server-side API. For now, let's do it the long way around with GET via sim_db_file. 

How much data is this file?

Reply:
[3d46d8f](https://github.com/radiasoft/sirepo/pull/5216/commits/3d46d8f319ea38d4abbf613dee0b4bb0cef4b516)
To repeat my earlier comment, not sure of this, although in any case other sim_data functions can use it

The file size is highly variable but anywhere from a megabyte to gigabytes; really as much as the user wants to generate.

Reply:
Obviated